modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
 separate upgrade strategies and also are used to control the cadence of updates.
 Channels are tied to a minor version of {product-title}. For instance,
 {product-title} {product-version} channels will never include an upgrade to a
-4.3 release. This ensures administrators make an explicit decision to upgrade to
+4.5 release. This ensures administrators make an explicit decision to upgrade to
 the next minor version of {product-title}. Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version}, which includes the upgrade from the previous 4.3
+release, has three upgrade channels to choose from:
+
+* candidate-{product-version}
+* fast-{product-version}
+* stable-{product-version}
+
+The upgrade channels contain two types of updates:
+
+.  General Availability Software (or GA) - These versions of {product-title} are
+fully supported and are considered production quality. You may upgrade to the general
+availability release from either of the fast and stable channels.
```

Comment:
This is misleading, because you may *also* be able to update to GA from an RC release.  But it matches [the existing KB article][1], so I would rather not evolve this language in this PR (we can improve the wording in follow-up work).

[1]: https://access.redhat.com/articles/4495171